x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/package_policies_table.tsx

```diff
@@ -109,29 +116,62 @@ export const PackagePoliciesTable: React.FunctionComponent<Props> = ({
         }),
         render: (value: string, packagePolicy: InMemoryPackagePolicy) => (
-          <EuiLink
-            title={value}
-            {...(canReadIntegrationPolicies
-              ? {
-                  href: getHref('edit_integration', {
-                    policyId: agentPolicy.id,
-                    packagePolicyId: packagePolicy.id,
-                  }),
-                }
-              : { disabled: true })}
-          >
-            <span className="eui-textTruncate" title={value}>
-              {value}
-            </span>
-            {packagePolicy.description ? (
-              <span>
-                &nbsp;
-                <EuiToolTip content={packagePolicy.description}>
-                  <EuiIcon type="help" />
-                </EuiToolTip>
-              </span>
-            ) : null}
-          </EuiLink>
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem data-test-subj="PackagePoliciesTableName" grow={false}>
+              <EuiLink
+                title={value}
+                {...(canReadIntegrationPolicies
+                  ? {
+                      href: getHref('edit_integration', {
+                        policyId: agentPolicy.id,
+                        packagePolicyId: packagePolicy.id,
+                      }),
+                    }
+                  : { disabled: true })}
+              >
+                <span className="eui-textTruncate" title={value}>
+                  {value}
+                </span>
+                {packagePolicy.description ? (
+                  <span>
+                    &nbsp;
+                    <EuiToolTip content={packagePolicy.description}>
+                      <EuiIcon type="help" />
+                    </EuiToolTip>
+                  </span>
+                ) : null}
+              </EuiLink>
+            </EuiFlexItem>
+            {canUseMultipleAgentPolicies &&
+              canReadAgentPolicies &&
+              canWriteIntegrationPolicies &&
+              getSharedPoliciesNumber(packagePolicy) > 1 && (
+                <EuiFlexItem grow={false}>
+                  <EuiToolTip
+                    content={
+                      <FormattedMessage
+                        id="xpack.fleet.agentPolicyList.agentsColumn.sharedTooltip"
+                        defaultMessage="This integration is shared by {numberShared} agent policies"
+                        values={{ numberShared: getSharedPoliciesNumber(packagePolicy) }}
+                      />
+                    }
+                  >
+                    <EuiText
+                      data-test-subj="PackagePoliciesTableSharedLabel"
+                      color="subdued"
+                      size="xs"
+                      className="eui-textNoWrap"
+                    >
+                      <FormattedMessage
+                        id="xpack.fleet.fleetServerSetup.sharedText"
+                        defaultMessage="Shared"
```

Comment:
is the i18n correct? `fleetServerSetup`?

please also force a space between the text and icon with `{' '}`